data/block/metaBlock.go

```diff
@@ -566,3 +566,14 @@ func (m *MetaBlock) CheckFieldsForNil() error {
 
 	return nil
 }
+
+// GetPreviousAggregatedSignatureAndBitmap returns the previous aggregated signature and the previous pubkeys bitmap
+func (m *MetaBlock) GetPreviousAggregatedSignatureAndBitmap() ([]byte, []byte) {
```

Comment:
we will need to do the same for metachain (equivalent proofs), otherwise we will still need to have a confirmation block on metachain.

Reply:
pushed

Reply:
not a strong opinion but maybe we could return the proof that holds them both
can stay like this for now, but if we realize that it will simplify implementation then we can do the switch

Reply:
sure.. added new method with []byte params in order to avoid cyclic imports while using block.Proof on the main HeaderHandler interface